wiki/en/en-Installation-for-Linux.md

```diff
@@ -102,9 +112,9 @@ Restart JACK to take any new settings
 
 ### Start Jamulus
 
-Open Jamulus e.g. via your command shell. If you use the shell and compiled Jamulus yourself, execute the command `Jamulus` (with a capital 'J') or if you used the .deb package with a lowercase j. This will start the Jamulus Client.
+From the command line, execute the command `jamulus`, or if you compiled from source `Jamulus` with a (with a capital 'J'). This will start the Jamulus Client.
 
-If you get problems with sound breaking up (in particular XRUN errors reported by JACK/QjackCtl) try setting bigger values (e.g. 256 frames or 3 periods) in step 3 above. Lower ones (e.g. 64 frames) could bring better performance but maybe more sound problems. See the [troubleshooting page](Client-Troubleshooting) otherwise.
+If you get problems with sound breaking up (in particular XRUN errors reported by JACK/QjackCtl) try setting bigger values (e.g. 256 frames or 3 periods). Lower ones (e.g. 64 frames) could bring better performance but maybe more sound problems. See the [troubleshooting page](Client-Troubleshooting) otherwise.
```

Comment:
> better performance

Should really say "lower latency" -- better performance is very vague.  What's "better" about more XRUNs, if that's what it causes someone?  Indeed, it's more that it requires a higher specification system - i.e. demands better performance, rather than provides it.

Anyway... for another time :).